### ISSUE TITLE
Is your code sending images taken by cams to a server ? 

### ISSUE THREAD

**Issue by Xeorox** [RodolfoFerro/pose-estimation#1]

**Comment by RodolfoFerro** [comment]
Yes, but I'm thinking to rollback to a functional version writing the outputs to a JSON file. 

I think that you can check out this version: https://github.com/RodolfoFerro/pose-estimation/tree/c4526e558e57551b9d2f154ead6cdf07c5b0416e

I haven't touched this code for a while, but if you're able to test it and it works as expected, let me know and I can rollback to this version.
